src/components/ArticleList.jsx

```diff
@@ -4,12 +4,20 @@ import Grid from "@mui/material/Unstable_Grid2";
 import Loading from "./Loading";
 import { useState } from "react";
 
-export default function ArticleList({ articleList }) {
+export default function ArticleList({ articleList, topic }) {
```

Comment:
If the articlesList isn't used in any other components, it might make sense to bring in further down the component tree and do the API call in this component 🤔 
I suppose you would need to pass more info via props to this component if you did this